fwoa-tools/src/migrationVerify.test.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+jest.mock('./migrationUtils', () => ({
+    checkConfiguration: () => {},
+    getFhirClientSMART: () => {
+        //eslint-disable-next-line
```

Comment:
Nit: specify which rule is disabled here

Reply:
updated